### PR TITLE
Chore/update nitro d09bae4

### LINKED ISSUE #279: bug: Do you want the app "Nitro" to accept incoming network requests?
**Describe the bug**
Users shouldn't have to know what Nitro is. We should change the copy to say "Jan" instead.

**To Reproduce**
1. Download model
2. Go to My Models
3. Click `Start`
4. Get the following alert, which is jarring because users will be like: "wut is Nitro? This app is called Jan"

**Expected behavior**
Quick fix: say Jan instead

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Mac M1, Sonoma
 - Version: v0.1.3

**Additional context**

### PR DESCRIPTION
**Pull request by vuonghoainam**
- [x] Update mac, windows, linux  Nitro to https://github.com/janhq/nitro/releases/tag/35-add-github-action-for-nitro-build
- [x] Update config file for new nitro
- [x] Add linux cuda instead of linux amd64 alone

Fix #279

### CODE REVIEW

File: electron/core/plugins/inference-plugin/module.js [hunk 1]
```diff
@@ -53,10 +53,10 @@ async function initModel(product) {
   if (process.platform === "win32") {
     binaryName = "nitro.exe";
   } else if (process.platform === "darwin") { // Mac OS platform
-    binaryName = process.arch === "arm64" ? "nitro" : "nitro_mac_intel";
+    binaryName = process.arch === "arm64" ? "nitro_mac_amd64" : "nitro_mac_amd64";
```
Comment (louis-jan) [inline electron/core/plugins/inference-plugin/module.js:56]:
```suggestion
    binaryName = process.arch === "arm64" ? "nitro_mac_arm64" : "nitro_mac_amd64";
```

File: electron/core/plugins/inference-plugin/nitro/config/config.json [hunk 2]
```diff
@@ -1 +1,13 @@
-{"custom_config": {"llama_model_path":"","ctx_len":2048,"ngl":100}}
+{
+	"listeners": [
+		{
+			"address": "0.0.0.0",
+			"port": 3928
+		}
+	],
+	"custom_config": {
+		"llama_model_path": "/Users/alandao/Documents/codes/nitro.cpp_temp/models/llama2_7b_chat_uncensored.Q4_0.gguf",
```
Comment (louis-jan) [inline electron/core/plugins/inference-plugin/nitro/config/config.json:9]:
```suggestion
		"llama_model_path": "",
```

File: electron/core/plugins/inference-plugin/nitro/config/config.json [hunk 3]
```diff
@@ -1 +1,13 @@
-{"custom_config": {"llama_model_path":"","ctx_len":2048,"ngl":100}}
+{
+	"listeners": [
+		{
+			"address": "0.0.0.0",
+			"port": 3928
```
Comment (louis-jan) [inline electron/core/plugins/inference-plugin/nitro/config/config.json:5]: Update `index.js` for the corresponding port

**Review by louis-jan — APPROVED** [review]
💯

**Review by hiento09 — APPROVED** [review]

**Pull request merged**